CRM/Case/Form/Search.php

```diff
@@ -381,6 +381,11 @@ public function fixFormValues() {
 
     $cid = CRM_Utils_Request::retrieve('cid', 'Positive', $this);
     if ($cid) {
+    $relc = CRM_Utils_Request::retrieve('relatedCases', 'Positive', $this);
+      if ($relc) {
+        $this->_formValues['contact_id'] = CRM_Case_BAO_Case::getOrganizationRelatedCaseIds($cid);
```

Comment:
Please double check this would work, if assigning an array of contacts to $this->_formValues['contact_id'] works as you'd expect or if something else would need to be done. See comment below on Page_View_Summary class, where it certainly won't work.